urbanopt/geojson-modelica-translator
# Cli expansion

#### Any background context you want to provide?
Expands cli to build and run the model
#### What does this PR accomplish?
adds 2 new cli commands
#### How should this be manually tested?
`poetry run uo_des -h` to see available commands. Try them all out.
- Ensure the `run-model` command works as expected
    - broken as of March 8, 2021. Internal gmt modelica variable names don't get found
#### What are the relevant tickets?
#### Screenshots (if appropriate)
